src/pages/RepoPage/FailedTestsTab/FailedTestsPage/FailedTestsTable/FailedTestsTable.tsx

```diff
@@ -266,7 +275,14 @@ const FailedTestsTable = () => {
     )
   }, [testData?.testResults])
 
-  const columns = useMemo(() => getColumns(!!hideFlakeRate), [hideFlakeRate])
+  const columns = useMemo(
+    () =>
+      getColumns({
+        hideFlakeRate: !!hideFlakeRate,
```

Comment:
the !! seems unnecessary. This could just be `hideFlakeRate,` right?

Reply:
Good point, I didn't really think about it but yep the !! is unnecessary